### PR TITLE
HowToUse Added

### PR DESCRIPTION
**Pull request by vipulsodha**
Merge how to use Documentation

### CODE REVIEW

**Comment by chetandhembre** [comment]
It will be nice if you create folder and add .eslint and how to use doc in it .. name folder as ``Node.JS style guide``

File: HowToUse.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## The following are the steps how to use Eslint.
+
+1. Include the .eslintrx file in your root folder.
+
+2. Download the Esling npm using 'npm install eslint'
+
+3. Add the following in your package.json
+  '"scripts": {
```
Comment (aps-sids) [inline HowToUse.md]: Use code block styles for code snippets in markdown files.

**Comment by vipulsodha** [comment]
ok

**Comment by chetandhembre** [comment]
@vipulsodha , @aps-sids  is right. @aps-sids  can you help vipul in this ?  It will be nice

**Comment by chetandhembre** [comment]
@vipulsodha  thanks . only thing remaining is folder . It will be nice if you get it done

**Pull request merged**